src/CUITeTest/SearchConfigurations/BaseConfiguratorTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CUITe.SearchConfigurations;
+using Microsoft.VisualStudio.TestTools.UITesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CUITeTest.SearchConfigurations
+{
+    using System;
```

Comment:
I'm in the process of trying to upgrade to Win10, and it isn't going very well since I am typing this from my tablet :-)

I can look at the settings when I manage to upgrade, but it shouldn't stop you from merging the PR. The code looks really good, let's get it to master.